agent/tests/test_lib_aws_sdk_php.c

```diff
@@ -211,6 +207,7 @@ static void test_nr_lib_aws_sdk_php_sqs_parse_queueurl() {
    */
   nr_segment_message_params_t message_params = {0};
   nr_segment_cloud_attrs_t cloud_attrs = {0};
+  char modifiable_string[512];
```

Comment:
Can you use `AWS_QUEUEURL_LEN_MAX` instead of "512"?

Reply:
Please see: [43fa2b8](https://github.com/newrelic/newrelic-php-agent/pull/1003/commits/43fa2b888d6583004d8159325270f1de6c0e16e9)